src/main/frontend/dicts.cljs

```diff
@@ -617,14 +616,6 @@
         :open-a-directory "Öffne ein lokales Verzeichnis"}
 
    :fr {:help/about "A propos de Logseq"
-        :on-boarding/demo-graph "This is a demo graph, changes will not be saved until you open a local folder."
```

Comment:
So the demo page's translation is gone?

Reply:
No behavior changed as this string is no different than the english key. When tongue doesn't find a key in a locale, it falls back to :en. I removed this and others because they aren't valid translations. I only did this for languages I'm familiar with (es, fr, pt*) as there are times when en and another language are the same. I used the `bb lang:duplicates` task to find these. Some of this is mentioned in the doc